test/max-asserts.js

```diff
@@ -1,80 +1,62 @@
 import test from 'ava';
 import avaRuleTester from 'eslint-ava-rule-tester';
 import rule from '../rules/max-asserts';
+import testCaseBuilder from './helpers/test-case';
 
 const ruleTester = avaRuleTester(test, {
 	env: {
 		es6: true
 	}
 });
 
-const errors = [{ruleId: 'max-asserts'}];
-const header = 'const test = require(\'ava\');\n';
+const ruleId = 'max-asserts';
+const basicTestCase = testCaseBuilder(ruleId);
+const lowerTestCase = testCaseBuilder(ruleId, [2]);
```

Comment:
Reading the code here I have no what the `[2]` means. I think the API could make it more readable.